### PR TITLE
fake a /workspace from cloud-builders/bazel image

### PR DESCRIPTION
**Pull request by krzyzacy**
because of https://github.com/GoogleCloudPlatform/cloud-builders/blob/master/bazel/Dockerfile#L52

so... will this work? :-)

/assign @ixdy @fejta @BenTheElder @Kashomon

### CODE REVIEW

**Comment by ixdy** [comment]
/shrug
/meow
/lgtm

We might want to upstream a better fix, too.

**Comment by krzyzacy** [comment]
(made https://github.com/GoogleCloudPlatform/cloud-builders/pull/385 to fix upstream)

File: config/jobs/GoogleCloudPlatform/k8s-cluster-bundle/k8s-cluster-bundle.yaml [hunk 1]
```diff
@@ -27,3 +33,9 @@ postsubmits:
         args: 
         - "test"
         - "..."
+        volumeMounts:
```
Comment (BenTheElder) [inline config/jobs/GoogleCloudPlatform/k8s-cluster-bundle/k8s-cluster-bundle.yaml]: make a preset?

Comment (krzyzacy) [reply config/jobs/GoogleCloudPlatform/k8s-cluster-bundle/k8s-cluster-bundle.yaml]: nah... will remove after upstream fix get in

**Pull request merged**